CONTRIBUTING.md

```diff
@@ -23,6 +49,81 @@ For feature requests, please:
 - Explain your motivation for the feature
 - List any alternatives you've considered
 
+## Branch Rules
+
+- `main` - Production-ready code, protected branch
+- `develop` - Integration branch for features
+- `feature/*` - New features
+- `release/*` - Release preparation
+- `bugfix/*` - Bug fixes for development
+- `hotfix/*` - Emergency production fixes
+
+## Version Control
+
+- Semantic versioning: `vMAJOR.MINOR.PATCH`
+- Release tags on `main` branch only
+- Package versions match git tags
+
+## Release Process
+
+week one for `release/v4.1.0`
```

Comment:
can you add the proposed release cycle duration - it is weekly release, bi-weekly release or monthly release?

I guess project at this scale weekly / bi-weekly works great.

Reply:
with this many people and not a lot of structure, short cycles will be frustrating. I think month cycles are prob the shortest that will work well. I am willing to try shorter if the maintainers think they can support it.